MAC_SETUP.md

```diff
@@ -97,35 +101,76 @@ test their code.
     ~/.pyenv/shims/python
     ```
 
+## Managing python versions using Pyenv global
+Pyenv allows you to configure the priority order for your python installs.
 
-## Using pyenv and pyenv-virtualenv to manage your Python versions
+```
+pyenv global 3.8.1 3.7.6 3.6.10 3.5.9 2.7.17
+```
 
-1.  Change to the desired source directory.
+This will make python and python3 point to Python 3.8.1. python2 will use 2.7.17. You can also further specify versions, such as python3.6 to use that version.
```

Comment:
This is a good explanation of something that wasn't entirely clear to me before 👍